src/js/stores/document.js

```diff
@@ -106,6 +112,7 @@ define(function (require, exports, module) {
          */
         _handleReset: function () {
             this._openDocuments = {};
+            this._layerIDToStateListenner = {};
```

Comment:
The `@type` annotation above indicates that this should be a `Map.<number, function>`, but here it's initialized to an `Object.<number, function>`. I'm generally in favor of using `Map` for lookup tables like this instead of `Object`.